src/commonMain/kotlin/teksturepako/pakku/cli/cmd/Diff.kt

```diff
@@ -15,43 +17,88 @@ class Diff : CliktCommand("Diff projects in modpack")
     private val newPathArg by argument("path")
     private val markdownDiffOpt by option("--markdown-diff", metavar = "<path>", help = "Export a `.md` file formatted as a diff code block")
     private val markdownOpt by option("--markdown", metavar = "<path>", help = "Export a `.md` file formatted as regular markdown")
+    private val detailedUpdate: Boolean by option("--detailed-update", help = "Gives detailed information on which mods were updated").flag()
```

Comment:
You could put the options and the help on new lines like it is done in the `set` cmd.